### ISSUE TITLE
Borg stun arm and flashes use old instant/hard stun mechanics, and not new stun mechanics.

### ISSUE THREAD

**Issue by Robustitron** [tgstation/tgstation#52525]
## Testmerges:
#51459: 'Baystruments - I had two tgstation forks for some reason and had to delete one and that nuked the last PR' by silicons at commit b9bf1a03c4
#52425: 'Adds Runechat Subsystem - Deprecates Use of Timers for Runechat' by bobbahbrown at commit 26371a1871
#52413: 'Overlay lighting component' by Rohesie at commit aa83789478

## Reproduction:
1) Be a human, get hit with a stun arm by a rogue/hacked borg
2) Be a borg or human, get hit by a flash
3) Notice in both scenarios you get an instant, long lasting hard-stun, a one click lose.

Basically human vs rogue borg stun mechanics are old fashioned one-click-win stuns for either side, which isn't consistent with how the rest of the game works nowadays, even the traitor stun items like the energy crossbow aren't capable of a one-click win.

Idk if its intentional that borg arms and flashes keep working the old fashioned way but here I am pointing out that maybe someone forgot to update them.

**Comment by Kaffe-work** [comment]
and i would like it no other way <3

**Comment by msgerbs** [comment]
Flashes in general (borg or otherwise) are still instant. The stun arms should probably be changed to work like stunbatons though.
